jax_triton/triton_lib.py

```diff
@@ -157,21 +163,49 @@ def aval_size_bytes(aval):
 
 
 @dataclasses.dataclass
-class PtxCompilationResult:
-  ptx: str
+class CompilationResult:
+  ptx_or_hsaco: str
```

Comment:
Shall we just call it `binary` or similar?

Reply:
Done